src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -878,7 +884,7 @@ boolean forkGitHub(Channel channel, User sender, String owner, String repo, Stri
             } catch (IOException e) {
                 // we started seeing 500 errors, presumably due to time out.
                 // give it a bit of time, and see if the repository is there
-                System.out.println("GitHub reported that it failed to fork "+owner+"/"+repo+". But we aren't trusting");
+                LOGGER.warn("GitHub reported that it failed to fork {}/{}. But we aren't trusting", owner, repo);
```

Comment:
"But we aren't trusting" what? This phrase finishes abruptly (out of this PR scope, just noticing)